solutions/standard/main.tf

```diff
@@ -120,6 +119,65 @@ module "elasticsearch" {
   enable_elser_model            = var.enable_elser_model
 }
 
+#create a service authorization between Secrets Manager and the target service (Elastic Search)
+resource "ibm_iam_authorization_policy" "policy" {
```

Comment:
The name is consumer visible. To avoid confusion can we avoid `policy` as a name. Maybe `secrets_manager_key_manager`.

So the full name will be `ibm_iam_authorization_policy. secrets_manager_key_manager`.

This may not be the best name.